sycl/doc/extensions/GroupAlgorithms/SYCL_INTEL_group_sort.asciidoc

```diff
@@ -241,77 +377,203 @@ position `i` in the ordered range.
 [source,c++]
 ----
 namespace sycl::ext::oneapi {
-  template <typename Group, typename Ptr>
-  void joint_sort(Group g, Ptr first, Ptr last); // (1)
 
-  template <typename Group, typename Ptr, typename Compare>
-  void joint_sort(Group g, Ptr first, Ptr last, Compare comp); // (2)
+  // Only available if GroupExecutor was created with a work-group and some associated scratch space

```

Comment:
To match the SYCL specification, this comment should be attached directly to each function it applies to.  As written, a reader might think this comment only applies to the first overload of `joint_sort` accepting a `GroupExecutor` (which I don't think is your intent).

Reply:
OK if it makes things clear

Reply:
Maybe it's better to write it after the code, not as a C++ comment inside the code?